From 5ca71ddc69ebb3696b63d260e15341653ddef4c0 Mon Sep 17 00:00:00 2001
From: Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>
Date: Fri, 18 Nov 2011 08:49:25 +0100
Subject: [PATCH] libubacktrace: use .so.$(ABI_VERSION)

Thanks to William Pitcock for noticing

Signed-off-by: Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>
(cherry picked from commit 69cba61dd27f2a9c4c34f596ed9d1b7cd0441f74)
---
 Rules.mak |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Rules.mak b/Rules.mak
index fe06c24..96074e5 100644
--- a/Rules.mak
+++ b/Rules.mak
@@ -118,7 +118,7 @@ export MAJOR_VERSION MINOR_VERSION SUBLEVEL VERSION ABI_VERSION LC_ALL
 
 LIBC := libc
 SHARED_LIBNAME := $(LIBC).so.$(ABI_VERSION)
-UBACKTRACE_DSO := libubacktrace.so.$(MAJOR_VERSION)
+UBACKTRACE_DSO := libubacktrace.so.$(ABI_VERSION)
 ifneq ($(findstring  $(TARGET_ARCH) , hppa64 ia64 mips64 powerpc64 s390x sparc64 x86_64 ),)
 UCLIBC_LDSO_NAME := ld64-uClibc
 ARCH_NATIVE_BIT := 64
-- 
1.7.8

